Source/stAiffWindow.cpp

```diff
@@ -113,8 +113,15 @@ void stAiffWindow::Thread()
 		for (int c = 0 ; c< (2 * sbufs); c++){
 
 			//max, min and clipping
-			if (sbuf[c] > max) max = sbuf[c];if (sbuf[c] < min) min = sbuf[c];
-			if (sbuf[c] > 0.999) sbuf[c] = 0.999; else if (sbuf[c] < -0.999) sbuf[c] = -0.999;
+			if (sbuf[c] > max)
+				max = sbuf[c];
+
+			if (sbuf[c] < min)
+				min = sbuf[c];
+
+			if (sbuf[c] > 0.999)
+				sbuf[c] = 0.999; else if (sbuf[c] < -0.999) sbuf[c] = -0.999;
```

Comment:
well you could fix the style of that else if as well while you're at it

Reply:
Will do, although this commit was primarily focusing on the fact that the program could not be compiled, rather than the code style.

I will follow up with another pull request afterwards.